### ISSUE TITLE
[EN] [Instance - Cata] Grim Batol - Erudax 

### ISSUE THREAD

**Issue by tincho388** [Atlantiss/BugTracker#4558]
At the ecounter with Erudax, you can pull him out of the room and avoid the adds. I don't know how it's on retail but i think it shouldn't work this way :P. 

I will try to add a screenshot today. 

Best Regards.

**Comment by xolon** [comment]
that's an exploit, but you don't need to do it if you can kill the adds fast.
on the second half i would like to say that sometimes adds are impossible, sometimes they spawn/go thru the wall to the eggs and you can't do anything but just to wipe from whelps.

**Comment by tincho388** [comment]
Yes i know I did Grim Batol lot of time the right way, but the other day i was in group with a tank that told us to stay outside, for curiosity i agreed and that is what happened. Is there a way to fix it? or is a retail problem?.

**Comment by zano354** [comment]
And sometimes the adds that spawn ignore any stun or interupt that they take, which is annoying.

**Comment by Aival94** [comment]
The adds not spawned is a blizzlike bug which i am not sure if get fixed after cataclysm(http://www.mmo-champion.com/threads/852371-Bug-exploit-on-Erudax-in-Grim-Batol).
About the adds ignore the stun/ccs i have never had such a problem and some ccs should not work on them so i believe this report should be closed.
